include/asm/format.h

```diff
@@ -0,0 +1,64 @@
+/*
+ * This file is part of RGBDS.
+ *
+ * Copyright (c) 2020, RGBDS contributors.
+ *
+ * SPDX-License-Identifier: MIT
+ */
+
+#ifndef RGBDS_FORMAT_SPEC_H
+#define RGBDS_FORMAT_SPEC_H
+
+#include <stdint.h>
+#include <stdbool.h>
+
+#define MAXSTRLEN	255
```

Comment:
At some point I needed it in format.h due to which .c files were using it and to avoid mutually recursive `#include`s, but apparently that's no longer the case.